ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__multi-nightly.yaml

```diff
@@ -2422,6 +2422,52 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone
+- as: gcp-ipi-xpn-mini-perm-dns-project-amd-f28-destructive
+  cron: 14 3 19 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_SERVICE_ACCOUNT: jiwei-worker-sa@openshift-qe.iam.gserviceaccount.com
+      CONTROL_PLANE_SERVICE_ACCOUNT: jiwei-control-plane-sa@openshift-qe.iam.gserviceaccount.com
+      PRE_CREATE_PRIVATE_ZONE: "no"
+      PRIVATE_ZONE_PROJECT: openshift-qe-regional
```

Comment:
In this kind of cluster: `IPI XPN + byo-hosted-zone`, we'll only set the private DNS zone to the PRIVATE_ZONE_PROJECT, and the public DNS records still would be created in the Host project, right?

Reply:
> IPI XPN + byo-hosted-zone, we'll only set the private DNS zone to the PRIVATE_ZONE_PROJECT

Right.

> the public DNS records still would be created in the Host project, right?

No. For a public cluster, the base domain is expected in the service project (rather than the host project).

Reply:
ack, thanks